pkg/apis/gcp/validation/shoot.go

```diff
@@ -57,7 +57,9 @@ func validateVolume(vol *core.Volume, fldPath *field.Path) field.ErrorList {
 	}
 	if vol.Type != nil && *vol.Type == worker.VolumeTypeScratch {
 		allErrs = append(allErrs, field.Invalid(
-			fldPath.Child("type"), worker.VolumeTypeScratch, fmt.Sprintf("type %s is not allowed as boot disk", worker.VolumeTypeScratch)))
+			fldPath.Child("type"),
+			worker.VolumeTypeScratch,
+			fmt.Sprintf("type %s is not allowed as boot disk", worker.VolumeTypeScratch)))
```

Comment:
```suggestion
			fldPath.Child("type"),
			worker.VolumeTypeScratch,
			fmt.Sprintf("type %s is not allowed as boot disk", worker.VolumeTypeScratch),
		))
```
Just a suggestion, keeps future diffs smaller. Also seems more consistent to me, but I realise that this was already done this way before.